engines/saga/render.cpp

```diff
@@ -253,7 +254,24 @@ void Render::addDirtyRect(Common::Rect r) {
 	} else \
 		_system->copyRectToScreen(_vm->_gfx->getBackBufferPixels(), _vm->_gfx->getBackBufferWidth(), x, y, w, h)
 
+void Render::maskSplitScreen() {
```

Comment:
Can you explain what is the split screen feature that is needed for the ECS version?
Please add a comment to explain what the feature does, and also add a comment here, explaining the purpose of this function. Thanks! :)

Reply:
Done above setSplitScreen